pkg/reconciler/pipelinerun/pipelinerun.go

```diff
@@ -1112,63 +1192,6 @@ func addMetadataByPrecedence(metadata map[string]string, addedMetadata map[strin
 	}
 }
 
-// getFinallyTaskRunTimeout returns the timeout to set when creating the ResolvedPipelineTask, which is a finally Task.
```

Comment:
I'm so happy to see these functions go away :D

Reply:
same here, thanks @abayer 🎉